test/pkg/integration/integration.go

```diff
@@ -57,25 +65,33 @@ func NewPodExec(config rest.Config, clientset *kubernetes.Clientset) *PodExec {
 func (p *PodExec) PodCopyFile(src string, dst string, containername string) (*bytes.Buffer, *bytes.Buffer, *bytes.Buffer, error) {
 	var in, out, errOut *bytes.Buffer
 	var ioStreams genericclioptions.IOStreams
-	for {
+	for count := 0; ; count++ {
 		ioStreams, in, out, errOut = genericclioptions.NewTestIOStreams()
 		copyOptions := kubectlcp.NewCopyOptions(ioStreams)
 		copyOptions.Clientset = p.Clientset
 		copyOptions.ClientConfig = p.RestConfig
 		copyOptions.Container = containername
 		err := copyOptions.Run([]string{src, dst})
 		if err != nil {
-			if !errors.Is(err, io.EOF) {
-				return nil, nil, nil, fmt.Errorf("Could not run copy operation: %v", err)
+			if !shouldRetry(count, err) {
+				return nil, nil, nil, fmt.Errorf("could not run copy operation: %v", err)
 			}
 			time.Sleep(10 * time.Second)
+			count = count + 1
```

Comment:
We have count++ at line 68, is this line still needed?

Reply:
you are right!